bin/codegen

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash -e
 
-CURRENT_DIR=$($(dirname $0)/abspath)
```

Comment:
this script doesn't work, but I don't think it's your changes ...

Reply:
new issue for this then?

Reply:
https://github.com/cyberark/secretless-broker/issues/663